src/js/components/List/doc.js

```diff
@@ -47,6 +47,13 @@ export const doc = List => {
 
   DocumentedList.propTypes = {
     ...genericProps,
+    action: PropTypes.func.description(
+      `The interactive element property for the user to be able to pass a 
+      component within list. It should be passed with an item and index of an
+      array and return a react element.
+      \`action = ({item, index}) => <Content />\`
+    />`,
```

Comment:
Thinking about this description some more, and the element doesn't have to be interactive, it could just be a component rendered on the cell. So maybe a different phrasing such as:
"Accepts a function that allows for a custom rendering of a component, it should be passed with an item and index of an array and return a react element."
Thoughts? Do you think it will be more accurate?

Reply:
yes it doesnt have to be interactive it can be a <box> for example or something else yes I think that sounds better

Reply:
Thank you for your patience and for brainstorming it through together...

Reply:
no problem it helped me better understand when I first looked I was also unsure about the action prop